english/call-a-doctor/let-met-out.md

```diff
@@ -0,0 +1,9 @@
+The doctor frowns. He shakes his head and scribbles something on a page inside a folder 
+thick with documents. When he looks up, you detect disappointment. The crow's feet 
+scratching at the corners of his eyes tell you that you've made this request before. 
+Maybe hundreds of times.
+
+"I can't do that," the doctor says. "Dr. Sakamoto will be here later on for another round 
+of shock treatments."
+
+[Sit and wait.](../../marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.